### PR TITLE
Feat: added command create device groups

### PR DESCRIPTION
**Pull request by maxwelbm**
- testing contract example for api main settings from edge
- command create device groups
- unit testing

### CODE REVIEW

File: pkg/api/edge_applications/main_settings_contract_api_test.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+			client := &http.Client{}
+			resp, err := client.Do(req)
+			if err != nil {
+				t.Errorf("error on request %v", err)
+				return
+			}
+			defer resp.Body.Close()
+
+			//if resp.StatusCode != tt.StatusExpected {
```
Comment (PatrickMenoti) [inline pkg/api/edge_applications/main_settings_contract_api_test.go:123]: remove commented code

**Review by PatrickMenoti — APPROVED** [review]

File: pkg/cmd/device_groups/create/.fixtures/response.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "results": {
+    "id": 2260,
+    "name": "Mob123il22e",
+    "user_agent": "Mobile|Android|iPhone"
+  },
+  "schema_version": 3
+}
```
Comment (ftoldato) [inline pkg/cmd/device_groups/create/.fixtures/response.json:8]: add new line at end of file

File: pkg/cmd/device_groups/create/.fixtures/request.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Mob123ile",
+  "user_agent": "Mobile|Android|iPhone"
+}
```
Comment (ftoldato) [inline pkg/cmd/device_groups/create/.fixtures/request.json:4]: add new line at end of file

File: messages/device_groups/messages.go [hunk 4]
```diff
@@ -30,6 +30,17 @@ var (
 	DeviceGroupsDescribeHelpFlag         = "Displays more information about the describe subcommand"
 	DeviceGroupsFileWritten              = "File successfully written to: %s\n"
 
+	// [ create ]
+	DeviceGroupsCreateUsage                 = "create [flags]"
+	DeviceGroupsCreateShortDescription      = "Creates a new device groups"
+	DeviceGroupsCreateLongDescription       = "Creates an device groups based on given attributes to be used in edge applications"
+	DeviceGroupsCreateFlagEdgeApplicationId = "Unique identifier for an edge application"
```
Comment (PatrickMenoti) [inline messages/device_groups/messages.go:37]: I think this flag is duplicated

**Review by gabriel-azion — APPROVED** [review]

**Pull request merged**